apps/web/src/app/[locale]/mypage/(github-custom)/FarmType.tsx

```diff
@@ -85,7 +85,12 @@ function FarmType() {
               loadingPersona={loadingPersona}
               selectPersona={selectPersona}
               onSelectPersona={onSelectPersona}
-              initSelectPersona={(list) => setSelectPersona(list)}
+              initSelectPersonas={(list) => {
+                // 현재 보여지는 펫들 처음부터 선택
+                const visiblePersonas = list.filter((persona) => persona.visible);
+                const visiblePersonaIds = visiblePersonas.map((persona) => persona.id);
+                setSelectPersona(visiblePersonaIds);
+              }}
```

Comment:
(마이너)

지금은 하는 동작이 복잡하지 않아서, 바로 체이닝 걸어도 좋을 거 같다는 갠적인 생각입니당
```suggestion
              initSelectPersonas={(list) => {
                // 현재 보여지는 펫들 처음부터 선택
                const visiblePersonaIds = list.filter((persona) => persona.visible).map((persona) => persona.id);
                setSelectPersona(visiblePersonaIds);
              }}
```